src/Symfony/Component/Process/Process.php

```diff
@@ -954,6 +954,16 @@ public function addErrorOutput(string $line)
         fseek($this->stderr, $this->incrementalErrorOutputOffset);
     }
 
+    /**
+     * Gets the last output time in microseconds
```

Comment:
this is in seconds, isn't it?

Reply:
We store the result of `microtime(true)` in the lastOutputTime field which "returns the current Unix timestamp with microseconds"

https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Process/Process.php#L289

BTW: the documentation of the fields are missing in this file.

Reply:
Yes, "with microseconds", but not "in microseconds". The value is stored in seconds, but with microsecond precision. That's why it's a float.

Reply:
Oh, I see. My bad.

Reply:
"Gets the last output time in seconds" would be enough IMHO as the return value is a `float`.